game.rb

```diff
@@ -0,0 +1,14 @@
+require_relative 'item'
+class Game < Item
+  def initialize(multiplayer, last_played_at, publish_date, archived)
+    super(publish_date, archived)
+    @multiplayer = multiplayer
+    @last_played_at = last_played_at
+  end
+
+  def can_be_archived?
+    return true if super && @last_played_at > 2
```

Comment:
[Optional]
It's a great job you overridden the implementation of `can_be_archived?`. But do you think `@last_played_at > 2` works properly? The date is input as a string and needs to be parsed. Please parse the date before the comparison. Kindly refer to [Date Class](https://ruby-doc.org/stdlib-2.5.1/libdoc/date/rdoc/Date.html).